app/models/generic_event/journey_change_vehicle.rb

```diff
@@ -0,0 +1,20 @@
+class GenericEvent
+  class JourneyChangeVehicle < GenericEvent
+    before_create :set_previous_vehicle_registration
+
+    details_attributes :vehicle_reg, :previous_vehicle_reg
```

Comment:
Could this be addressed with a versioned model instead rather than hand rolled versioning for specific attribute?

Reply:
Just talking this through with Sangeetha. I'd actually like to remove history in the event that belongs to the eventable if possible (or force the supplier to provide it/have the report work it out).

Reply:
Okay, killed the versioning. Client just supplies this value after they've broken down and get a vehicle replacement so they have the information we need for the JPC.

Reply:
Awesome, nice to keep it simple. Great work convincing people with this 👍